pkgs/development/python-modules/werkzeug/default.nix

```diff
@@ -1,30 +1,56 @@
-{ lib, stdenv, buildPythonPackage, fetchPypi
-, itsdangerous, hypothesis
-, pytestCheckHook, requests
+{ lib
+, stdenv
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, watchdog
+, dataclasses
 , pytest-timeout
-, isPy3k
+, pytest-xprocess
+, pytestCheckHook
  }:
 
 buildPythonPackage rec {
   pname = "Werkzeug";
-  version = "1.0.1";
+  version = "2.0.1";
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "6c80b1e5ad3665290ea39320b91e1be1e0d5f60652b964a3070216de83d2e47c";
+    sha256 = "0hlwawnn8c41f254qify5jnjj8xb97n294h09bqimzqhs0qdpq8x";
   };
 
-  propagatedBuildInputs = [ itsdangerous ];
-  checkInputs = [ pytestCheckHook requests hypothesis pytest-timeout ];
+  propagatedBuildInputs = [
+    watchdog
+  ] ++ lib.optionals (pythonOlder "3.7") [
+    dataclasses
+  ];
+
+  checkInputs = [
+    pytest-timeout
+    pytest-xprocess
+    pytestCheckHook
```

Comment:
I prefer to put ``pytestCheckHook`` first, but that's personal preference.

Reply:
Alphabetically sorted